deploy/cephcsi/image/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM ${BASE_IMAGE} as builder
 
 LABEL stage="build"
 
-ARG GOLANG_VERSION=1.13.9
+ARG GOLANG_VERSION=1.14.4
```

Comment:
This will be overridden by https://github.com/ceph/ceph-csi/blob/master/Makefile#L146 ie 1.13.8